src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -727,7 +727,7 @@ def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype
         return latents
 
     def _get_add_time_ids(
-        self, original_size, crops_coords_top_left, target_size, dtype, text_encoder_projection_dim=None
+        self, original_size, crops_coords_top_left, target_size, dtype, text_encoder_projection_dim=1280
```

Comment:
```suggestion
        self, original_size, crops_coords_top_left, target_size, dtype, text_encoder_projection_dim
```

Reply:
this may damage some external dependencies

Reply:
this may damage some external dependencies